docs/sources/tempo/release-notes/v2-7.md

```diff
@@ -209,7 +214,25 @@ max_spans_per_span_set
 This release disables gRPC compression in the querier and distributor for performance reasons. ([#4429](https://github.com/grafana/tempo/pull/4429))
 Our benchmark suggests that without compression, queriers and distributors use less CPU and memory.
 
+{{< admonition type="note" >}}
+While disabling gRPC compression improves performance, it may increase data usage and network traffic, which can impact Grafana Cloud billing, especially for larger clusters.
```

Comment:
```suggestion
While disabling gRPC compression improves performance, it will increase data usage and network traffic, which can impact cloud billing depending on your configuration.
```